### ISSUE TITLE
tkinter broken in python3.5 image

### ISSUE THREAD

**Issue by schultzmattd** [docker-library/python#145]
Hi everyone,

I ran into an issue importing the [tkinter module](https://docs.python.org/3.5/library/tkinter.html) from the python standard library with the 3.5 image:

```
➜  thermal_shift git:(master) ✗ docker run --rm -it python:3.5 bash
root@ed2aa6dfd036:/# python
Python 3.5.1 (default, May 12 2016, 19:06:45)
[GCC 4.9.2] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import tkinter
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.5/tkinter/__init__.py", line 35, in <module>
    import _tkinter # If this fails your Python may not be configured for Tk
ImportError: No module named '_tkinter'
```

Importing the module seems to work in python 3.4 though:

```
➜  thermal_shift git:(master) ✗ docker run --rm -it python:3.4 bash
root@555940af6c20:/# python
Python 3.4.5 (default, Aug  9 2016, 20:44:58)
[GCC 4.9.2] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import tkinter
>>>
root@555940af6c20:/# exit
```

Any thoughts on what might be amiss would be great. Thanks in advance!

**Comment by yosifkit** [comment]
I am unable to reproduce here.  How old is your 3.5 image? Have you done a docker pull recently?  This was fixed in #127 and #130, which would have been included in https://github.com/docker-library/official-images/pull/1958 and https://github.com/docker-library/official-images/pull/2005, respectively.

**Comment by schultzmattd** [comment]
That fixed it. Thanks very much for the help! So sorry for the silly mistake...
